### PR TITLE
add new php image number 7.3

### PR DESCRIPTION
**Pull request by fabienChaillou**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -227,6 +227,20 @@ php:
...
+      COMPOSER_VERSION: 1.8.5
+      MEMCACHED_VERSION: 3.1.3
+      MODD_VERSION: *MODD_VERSION
+      REDIS_VERSION: 4.3.0
+      SECURITY_CHECKER_VERSION: 5.0.3
+      SSH2_VERSION: 1.1.2
+      XDEBUG_VERSION: 2.7.2
+    cmd_test: *php_cmd_test
+    template_vars:
+      PHP_VERSION: 7.2.18-cli-alpine
```
Comment (mremi) [inline config.yml]: Fail

File: .travis.yml [hunk 2]
```diff
@@ -30,6 +30,7 @@ matrix:
     - env: IMAGE=node VERSION=12
     - env: IMAGE=php VERSION=7.1
```
Comment (mremi) [inline .travis.yml]: We can stop maintaining this version

Comment (fabienChaillou) [reply .travis.yml]: Ok I removed php version 7.1

File: config.yml (hunk 1 above)
Comment (fabienChaillou) [reply config.yml]: I change `PHP_VERSION` at 7.3.9

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 Versions
 ========
 
+2019-09-06
+----------
+
+* Add a new image PHP version 7.3
+* Remove support for PHP 7.1
```
Comment (bmaziere) [inline CHANGELOG.md]: should be in -DEV section i guess

File: php/Dockerfile.j2 [hunk 4]
```diff
@@ -23,7 +23,7 @@ RUN echo "Starting ..." && \
     echo "@edge-community http://nl.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
     apk add --update --upgrade alpine-sdk apk-tools@edge-main autoconf bash bzip2 cyrus-sasl-dev curl freetype-dev gettext git \
         gnu-libiconv@edge-community icu-dev@edge-main libjpeg-turbo-dev libmcrypt-dev libmemcached-dev libpng-dev libssh2-dev \
-        libxml2-dev make openssh-client patch postgresql-dev rsync tzdata && \
+        libxml2-dev libzip-dev make openssh-client patch postgresql-dev rsync tzdata && \
```
Comment (mremi) [inline php/Dockerfile.j2]: please revert

File: php/Dockerfile.j2 [hunk 5]
```diff
@@ -43,8 +43,8 @@ RUN echo "Starting ..." && \
     pecl install memcached-${MEMCACHED_VERSION} && \
     docker-php-ext-enable memcached && \
     docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql && \
-    pecl install ssh2-${SSH2_VERSION} && \
-    docker-php-ext-enable ssh2 && \
+    # pecl install ssh2-${SSH2_VERSION} && \
```
Comment (mremi) [inline php/Dockerfile.j2]: please add before `# waiting for https://github.com/php/pecl-networking-ssh2/pull/36 to be merged`

File: php/test.php [hunk 6]
```diff
@@ -142,16 +142,10 @@ private function checkExtensions()
 
         $errors             = array();
         $extensionsExpected = array(
-            'bcmath', 'exif', 'gd', 'iconv', 'intl', 'mbstring', 'pcntl', 'pdo_mysql',
-            'pdo_pgsql', 'redis', 'soap', 'sockets', 'xdebug', 'zip',
+            'apcu', 'bcmath', 'blackfire', 'exif', 'gd', 'iconv', 'intl', 'mbstring', 'memcached', 'pcntl', 'pdo_mysql',
+            'pdo_pgsql', 'pgsql', 'redis', 'soap', 'sockets', 'ssh2', 'xdebug', 'Zend OPcache', 'zip',
```
Comment (mremi) [inline php/test.php]: `ssh2` must be omitted until support is ok.

BTW the build should be down. @Mallear something is broken, Travis should detect that `php test.php` returned a failure (through the returned status code), could you please have a look?

File: php/Dockerfile.j2 (hunk 4 above)
Comment (fabienChaillou) [reply php/Dockerfile.j2]: PHP 7.3 version have to need `libzip-dev` extension if it enabled zip extension.
https://www.php.net/manual/en/zip.installation.php

File: php/test.php (hunk 6 above)
Comment (fabienChaillou) [reply php/test.php]: Ok remove the check if ssh2 extension enabled

File: CHANGELOG.md [hunk 7]
```diff
@@ -12,8 +12,8 @@ Versions
 * Lock scss_lint version to fix issue with Ruby version
 * Upgrade Node version: 6.17.1, 8.16.0
 * Upgrade NVM version: 0.34.0
-* Upgrade PHP version: 7.1.29, 7.2.18
-* Upgrade Composer version: 1.8.5
+* Upgrade PHP version: 7.1.29, 7.2.22
```
Comment (mremi) [inline CHANGELOG.md]: please remove 7.1.29

File: CHANGELOG.md [hunk 8]
```diff
@@ -37,6 +37,10 @@ Versions
 * Create image with azure CLI and terraform
 * Upgrade Golang version: 1.13
 * Upgrade Serverless version: 1.51
+* Add a new image PHP version 7.3
```
Comment (mremi) [inline CHANGELOG.md]: please move it before in order to group all php changes together

**Pull request merged**